toolchain/check/import_ref.cpp

```diff
@@ -2881,6 +2779,12 @@ static auto TryResolveInstCanonical(ImportRefResolver& resolver,
     case CARBON_KIND(SemIR::BindAlias inst): {
       return TryResolveTypedInst(resolver, inst);
     }
+    case CARBON_KIND(SemIR::BindingPattern inst): {
+      return TryResolveTypedInst(resolver, inst, inst_id);
+    }
+    case CARBON_KIND(SemIR::SymbolicBindingPattern inst): {
+      return TryResolveTypedInst(resolver, inst, inst_id);
+    }
```

Comment:
I'd prefer this list to be kept in alphabetical order. (And similarly for the definition of the corresponding types in `typed_insts.h`.) While I understand the desire to cluster these, any clustering we do is going to be a bit arbitrary, and make it harder to find specific cases and harder to maintain the list compared to a strictly alphabetical order.

Reply:
See #5401.